src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=8.1",
         "psr/container": "^1.1|^2.0",
         "symfony/deprecation-contracts": "^2.5|^3",
-        "symfony/service-contracts": "^2.5|^3.0",
+        "symfony/service-contracts": "^3.2",
```

Comment:
Ah, I forgot about this one.
Any other idea to account for https://github.com/symfony/symfony/pull/51832#pullrequestreview-1670798856?
Maybe we should skip the test case when contracts < 3.2 is loaded instead? (and throw in the code - or do nothing waiting for someone to hit this for-now hypothetical situation?)